Docs/IssueTemplate.md

```diff
@@ -0,0 +1,32 @@
+[provide general introduction to the issue logging and why it is relevant to this repository]
+
+## Context
+
+[provide more detailed introduction to the issue itself and why it is relevant]
+
+## Process
+
+[ordered list the process to finding and recreating the issue, example below]
+
+1. User goes to delete a dataset (to save space or whatever)
+2. User gets popup modal warning
+3. User deletes and it's lost forever
+
+## Expected result
+
+[describe what you would expect to have resulted from this process]
+
+## Current result
+
+[describe what you you currently experience from this process, and thereby explain the bug]
+
+## Possible Fix
+
+[not obligatory, but suggest fixes or reasons for the bug]
+
+
+## `name of issue` screenshot
+
+[if relevant, include a screenshot]
+
+----
```

Comment:
I would rather remove this.  It would be preferable to use the GitHub issues template instead.